src/Decred.BlockExplorer/TransactionRepository.cs

```diff
@@ -0,0 +1,221 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Dapper;
+using DcrdClient;
+using NDecred.Common;
+using Paymetheus.Decred.Script;
+using Paymetheus.Decred.Wallet;
+
+namespace Decred.BlockExplorer
+{
+    public interface ITransactionRepository
```

Comment:
I'll mark it here again. I mean, that it should be fixed everywhere: "one class/interface - one file"